src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
```

Comment:
The message is completely wrong, isn't it? As this deprecation was done 6 days ago, meaning in 2.7-dev: https://github.com/symfony/symfony/pull/13141 The message states version 2.3 already deprecated it.